core/vm/contracts.go

```diff
@@ -147,7 +150,30 @@ func init() {
 	}
 }
 
-// ActivePrecompiles returns the precompiles enabled with the current configuration.
+// Copy returns a copy of the precompiled contracts.
+func (p PrecompiledContracts) Copy() PrecompiledContracts {
+	c := make(PrecompiledContracts)
+	for k, v := range p {
+		c[k] = v
+	}
+	return c
+}
+
+// ActivePrecompiledContracts returns precompiled contracts enabled with the current configuration.
+func ActivePrecompiledContracts(rules params.Rules) PrecompiledContracts {
+	switch {
+	case rules.IsBerlin:
+		return PrecompiledContractsBerlin
```

Comment:
I think you lost `PrecompiledContractsCancun` in some rebase somewhere

Reply:
Fixed thanks!